src/main/java/monitoringsystemturbo/Presentation.java

```diff
@@ -120,7 +120,7 @@ private static void exportData(List<String> programNames, TrackingService tracki
     private static void saveProcessesToConfig(String[] processNames) throws FileNotFoundException {
         List<Application> listToSave = new ArrayList<Application>();
         for (String processName : processNames) {
-            listToSave.add(new Application(processName, "C://"));
+            listToSave.add(new Application(processName, "C://Users/kkal6/AppData/Local/Google/Chrome SxS/Application/chrome.exe"));
```

Comment:
maybe there is a way to customize it? Hardcoded path, especially with user id in code, is not a good idea. Consider using gradle if possible

Reply:
It's just presentation for client thus I could skip it, or maybe should I remove entire file?

Reply:
For now, let's leave it like this, we will remove it at the end